Various/amagalma_Implode items across tracks into takes (remove empty tracks).lua

```diff
@@ -0,0 +1,43 @@
+-- @description amagalma_Script: Implode items across tracks into takes (remove empty tracks)
+-- @author amagalma
+-- @version 1.0
+-- @about
+--   # Script: Implodes items across tracks into takes, removes empty tracks and inherits their height
+--
+
+-- @link http://stash.reaper.fm/30479/amagalma_Implode%20items%20across%20tracks%20into%20takes%20%28remove%20empty%20tracks%29.gif
```

Comment:
The empty line 7 marks the end of the header thus the `@link` at line 8 won't not be read. Also, I suggest using the `@screenshot` tag: it will show a button labeled a such rather than an unclear "Website".